src/main/java/appeng/me/GridConnection.java

```diff
@@ -24,7 +24,10 @@
 
 import net.minecraftforge.common.util.ForgeDirection;
 
+import appeng.api.exceptions.ExistingConnectionException;
 import appeng.api.exceptions.FailedConnection;
+import appeng.api.exceptions.SecurityConnectionException;
+import appeng.api.exceptions.NullNodeConnectionException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Wrong order for 'appeng.api.exceptions.NullNodeConnectionException' import. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)